fhir-parent/pom.xml

```diff
@@ -47,6 +47,7 @@
         <module>../fhir-model</module>
         <module>../fhir-registry</module>
         <module>../fhir-term</module>
+        <module>../fhir-term-graph</module>
```

Comment:
do you think terminology may end up having other implementations.  does fhir-term deserve a top  level folder?

Reply:
Although I can see the potential for plugging in other `FHIRTermServiceProvider` implementations via the service loader but I don''t think we need a top-level folder at this time.